### ISSUE TITLE
Migrate to ts

### ISSUE THREAD

**Issue by iamvikshan** [iamvikshan/amina#276]

**Comment by ShockAnWave** [comment]
so no more js?

**Comment by iamvikshan** [comment]
nah, the bot will still run on js, but at backed it is built with ts, it just makes my work harder to code, but get 100% assurance that there will be no runtime errors. technically you have to recompile the bot to js, which spits out js. dw, this is automated
